docs/ambiguity.md

```diff
@@ -7,9 +7,12 @@ Due to DST time changes, there is a possibility that a wall-clock time either do
 
 There are two mostly equivalent methods that accomplish this conversion: [`Temporal.DateTime.prototype.inTimeZone`](./datetime.html#inTimeZone) and [`Temporal.TimeZone.prototype.getAbsoluteFor`](./timezone.html#getAbsoluteFor).
 The `disambiguation` option to these methods controls what absolute time to return in the case of ambiguity:
-- `earlier` (the default): The earlier of two possible absolute times will be returned.
-- `later`: The later of two possible absolute times will be returned.
-- `reject`: A `RangeError` will be thrown.
+
+- `'compatible'` (the default): Acts like `'earlier'` for backward transitions like the start of DST in the Spring, and `'later'` for forward transitions like the end of DST in the Fall.
+  This matches the behavior of legacy `Date`, of libraries like moment.js, Luxon, and date-fns, and of cross-platform standards like [RFC 5545 (iCalendar)](https://tools.ietf.org/html/rfc5545).
```

Comment:
I'd move this part out of the parameter descriptions and into the paragraph below, so that this list of options stays brief.
(also in several other places below)

Reply:
done, and moved into a short paragraph about compatibility below the options.